modules/gitops-token-lifecycle-management.adoc

```diff
@@ -0,0 +1,26 @@
+// Module is included in the following assemblies:
+//
+// * accesscontrol_usermanagement/managing-local-users-in-argo-cd.adoc
+
+:_mod-docs-content-type: CONCEPT
+[id="gitops-token-lifecycle-management_{context}"]
+= Token lifecycle management
+
+The Argo CD Operator manages the lifecycle of local user tokens. Depending on the configuration, tokens can be renewed automatically or rotated manually.
+
+== Automatic renewal
+
+You can configure the Operator to automatically renew tokens before they expire. This ensures that automation and integrations that depend on the token continue to work without interruption.
+
+== Manual token rotation
+
+If you need to replace a token immediately, you can manually rotate it by deleting the user secret. The Argo CD Operator then generates a new token and updates the configuration. Run the following command to rotate a token:
+
+[source,terminal]
+----
+kubectl delete secret api-user-local-user -n argocd
+----
+
+== Disabling API keys
+
+You can disable API key generation for a local user to remove their access through tokens. Disabling API keys also cleans up the associated secrets and stops renewal timers.
```

Comment:
Should maybe clarify that disabling API key generation is done by setting the `apiKey` field to false.

Reply:
Fixed.